### ISSUE TITLE
Ubuntu 20.04 supported?

### ISSUE THREAD

**Issue by geosme** [gisgraphy/gisgraphy#62]
Wondering if Ubuntu 20.04 is supported or should maybe consider 18.04 instead for a self deployment?

**Comment by gisgraphy** [comment]
Yes the script will be available before end of year. It also requires to be compatible with postgres 10 and higher and postgis 2 or 3.-------

**Comment by geosme** [comment]
Thank you for your prompt reply.
So what is the recommended distro & version and proven with least problems?  Ubuntu 18.04 LTS server edition or better to deploy a desktop edition?

**Comment by gisgraphy** [comment]
16.04 works fine. There is no problems with any Ubuntu version. Just incompatible with postgis 2. And more. The compatible version is already ready but not yet available. I can comit the code and you can build it from source. I can also send a link to a snapshot version. David Masclet-------- Message original --------Objet : Re: [gisgraphy/gisgraphy] Ubuntu 20.04 supported? (#62)De : geosme À : gisgraphy/gisgraphy Cc : gisgraphy ,Comment 
Thank you for your prompt reply.
So what is the recommended distro & version and proven with least problems?  Ubuntu 18.04 LTS server edition or better to deploy a desktop edition?

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/gisgraphy/gisgraphy/issues/62#issuecomment-724044139",
"url": "https://github.com/gisgraphy/gisgraphy/issues/62#issuecomment-724044139",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by geosme** [comment]
Thanks. I can wait. 
Feel free to send a link to snapshot version to take a peak.
How long before you commit installer script you think and I assume it will be for 20.04? Under a month?

**Comment by gisgraphy** [comment]
Ok. Yes i can send a link. Everything is ready. Just need to find time 😁David Masclet-------- Message original --------Objet : Re: [gisgraphy/gisgraphy] Ubuntu 20.04 supported? (#62)De : geosme À : gisgraphy/gisgraphy Cc : gisgraphy ,Comment 
Thanks. I can wait.
Feel free to send a link to snapshot version to take a peak.
How long before you commit installer script you think and I assume it will be for 20.04? Under a month?

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/gisgraphy/gisgraphy/issues/62#issuecomment-724052040",
"url": "https://github.com/gisgraphy/gisgraphy/issues/62#issuecomment-724052040",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by miniatarule** [comment]
HI! Where is the snapshot link of the latest compiled version?

the version gisgraphy-5.0-beta3 that can be downloaded on your Website is the latest version?

**Comment by gisgraphy** [comment]
HiI have upload The latest version compatible with ubuntu 20 In the snapshot directory on the download serverLet me know if problem Regards David Masclet-------- Message original --------Objet : Re: [gisgraphy/gisgraphy] Ubuntu 20.04 supported? (#62)De : miniatarule À : gisgraphy/gisgraphy Cc : gisgraphy ,Comment 
HI! Where is the snapshot link of the latest compiled version?
the version gisgraphy-5.0-beta3 that can be downloaded on your Website is the latest version?

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/gisgraphy/gisgraphy/issues/62#issuecomment-786825439",
"url": "https://github.com/gisgraphy/gisgraphy/issues/62#issuecomment-786825439",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by miniatarule** [comment]
Could you share the direct link to the download folder please.

**Comment by miniatarule** [comment]
That is the link right?
https://download.gisgraphy.com/snapshots/

**Comment by gisgraphy** [comment]
YesDavid Masclet-------- Message original --------Objet : Re: [gisgraphy/gisgraphy] Ubuntu 20.04 supported? (#62)De : miniatarule À : gisgraphy/gisgraphy Cc : gisgraphy ,Comment 
That is the link right?
https://download.gisgraphy.com/snapshots/

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/gisgraphy/gisgraphy/issues/62#issuecomment-786844522",
"url": "https://github.com/gisgraphy/gisgraphy/issues/62#issuecomment-786844522",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by miniatarule** [comment]
Do you recommend install the latest version postgres, Postgis and java jvm 1.8?  at this page: https://www.java.com/en/download/manual.jsp
Sorry for all those question, I'm a windows guy and a .net programmer.
